src/lib/y2partitioner/widgets/lvm_pv_table.rb

```diff
@@ -0,0 +1,74 @@
+require "yast"
+
+require "cwm/table"
+
+require "y2partitioner/icons"
+require "y2partitioner/widgets/blk_devices_table"
+
+module Y2Partitioner
+  module Widgets
+    # Table widget to represent given list of Y2Storage::LvmLvs together.
+    class LvmPvTable < CWM::Table
+      include BlkDevicesTable
+
+      # @param pvs [Array<Y2Storage::LvmPv] devices to display
+      # @param pager [CWM::Pager] table have feature, that double click change content of pager
+      #   if someone do not need this feature, make it only optional
+      def initialize(pvs, pager)
+        textdomain "storage"
+        @pvs = pvs
+        @pager = pager
+      end
+
+      # table items. See CWM::Table#items
+      def items
+        @pvs.map do |pv|
+          device = pv.plain_blk_device
+          [
+            id_for_device(device), # use name as id
+            device.name,
+            device.size.to_human_string,
+            # TODO: dasd format use "X", check it
+            device.exists_in_probed? ? "" : _(BlkDevicesTable::FORMAT_FLAG),
```

Comment:
Actually, after checking the old partitioner behavior. This flag literally means whether the partition is going to be formatted with a real filesystem (not whether is going to be used as PV). Obviously, the column makes no sense in this context because something going to be used as PV is, by definition, not going to be formatted. But I guess we should keep the old behavior. So just wait for `BlkDevice#to_be_formatted?` to appear in yast-storage-ng (WIP).